packages/astro/test/test-utils.js

```diff
@@ -59,7 +59,8 @@ export async function loadFixture(inlineConfig) {
     build: (opts = {}) => build(config, { mode: 'development', logging: 'error', ...opts }),
     startDevServer: async (opts = {}) => {
       const devServer = await dev(config, { logging: 'error', ...opts });
-      inlineConfig.devOptions.port = devServer.port; // update port
+      config.devOptions.port = devServer.port; // update port
```

Comment:
`inlineConfig` is the reference passed into a test, but `config` is the reference used by `.fetch()`. Fixed a flakiness bug where `fetch()` had an old port.